src/test/java/dk/cngroup/lentils/repository/UserRepositoryTest.java

```diff
@@ -41,11 +41,16 @@ public void saveTeamCreateNewUser() {
 
     @Test
     public void adminUserIsImportedOnStartup() {
-        Assert.assertEquals(1, userRepository.count());
         List<User> users = userRepository.findAll();
         Assert.assertEquals("admin", users.get(0).getUsername());
```

Comment:
OPTIONAL: tady by melo byt neco jako: 
```java
User admin = users.stream.findAny(u -> u.getUsername().equals("admin"));
Assert.assertNotNull(admin);
```
protoze takto predpokladas, ze ho db vrati vzdy prvniho, ale to nemusi platit, kdyz nekdo prohodi id v tom init scriptu.